### PR TITLE
Fixed error in #12

### PR DESCRIPTION
**Pull request by darrenstahlmsft**
Oops, capitalization error on last PR. This fixes it @jstarks 

Signed-off-by: Darren Stahl <darst@microsoft.com>

### CODE REVIEW

File: file.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 //sys createIoCompletionPort(file syscall.Handle, port syscall.Handle, key uintptr, threadCount uint32) (newport syscall.Handle, err error) = CreateIoCompletionPort
 //sys getQueuedCompletionStatus(port syscall.Handle, bytes *uint32, key *uintptr, o **ioOperation, timeout uint32) (err error) = GetQueuedCompletionStatus
 //sys setFileCompletionNotificationModes(h syscall.Handle, flags uint8) (err error) = SetFileCompletionNotificationModes
-//sys timeBeginPeriod(period uint32) (n int32) = TimeBeginPeriod
+//sys timeBeginPeriod(period uint32) (n int32) = timeBeginPeriod
```
Comment (jstarks) [inline file.go]: This should be winmm.timeBeginPeriod

**Comment by darrenstahlmsft** [comment]
Seems to work as is, but I will update

**Comment by darrenstahlmsft** [comment]
Do we want this to be protected by and environment variable, or always enabled?

**Comment by jstarks** [comment]
Always enabled.

**Pull request merged**